components/GlobalFooter/GlobalFooter.tsx

```diff
@@ -1,15 +1,10 @@
-import { cnb } from 'cnbuilder';
 import { StanfordLogo } from '@/components/Logo';
 import { Container } from '@/components/Container';
 import { FlexBox } from '@/components/FlexBox';
 import * as styles from './GlobalFooter.styles';
 
-type GlobalFooterProps = {
-  className?: string;
-};
-
-export const GlobalFooter = ({ className }: GlobalFooterProps) => (
```

Comment:
Removed the props from Global Footer because we aren't passing any classname anyway